src/assets/recipeData.json

```diff
@@ -120,48 +120,48 @@
     "material": "Armor Trim",
     "craftable": false,
     "stat": "Armour",
-    "rarity": "Common",
+    "rarity": "common",
     "locations": [
       "FoC7",
       "O11",
       "SH5",
       "OM22",
-      "FoC1"
+      "FoCE1"
     ]

```

Comment:
let's drop the locations at all from the recipe data. Iw ill compute them based on battleData.json so there will be single source of truth